src/tasks/LibraryBuilder/LibraryBuilder.cs

```diff
@@ -85,6 +85,16 @@ public bool IsSharedLibrary
     /// </summary>
     public string? AssembliesLocation { get; set; }
 
+    /// <summary>
+    /// Determines whether or not assemblies are bundled into the library
+    /// </summary>
+    public bool BundlesResources { get; set; }
+
+    /// <summary>
+    /// An Item containing the bundled runtimeconfig.bin metadata populated by EmitBundle task
```

Comment:
Drop reference to EmitBundle task in comment (tasks are not strictly related), better to document what metadata needs to be included and what values they should include.

Is parameter of type ITaksItem[] since we want to handle an items list from msbuild? We will only handle one BundledRuntimeConfig, so maybe we should add that to comment as well as check and fail if you pass in more than one item, since we only handles the first anyways.